src/main/java/uk/gov/digital/ho/hocs/application/MessageContext.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.digital.ho.hocs.application;
+
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+public class MessageContext {
+
+    public static final String CORRELATION_ID_HEADER = "X-Correlation-Id";
+
+    public static final String USER_ID_HEADER = "X-Auth-UserId";
+
+    public static final String GROUP_HEADER = "X-Auth-Groups";
+
+    private static final String MESSAGE_ID_HEADER = "Message-Id";
+
+    private static final String TEAM_ID = "TEAM_ID";
+
+    private final RequestData requestData;
```

Comment:
Do we need both MessageContext and RequestionData? Could we remove RequestData as everything comes from the queue except the 1 endpoint which won't have any request data because it's being called from a cron job?

Reply:
I was in two minds about it - if a request comes through the API we'd want all of the logs to show who did what, even from a manual standpoint. We'd likely also maybe want to see the logs associated with the entire process vs with a singular message perhaps?